app/models/concerns/by_docket_date_distribution.rb

```diff
@@ -93,23 +93,25 @@ def ama_statistics
   def ama_distributed_cases_tied_to_ineligible_judges
     @appeals.filter_map do |appeal|
       appeal[:case_id] if HearingRequestDistributionQuery.ineligible_judges_id_cache
-        &.include?(hearing_judge_id(appeal[:case_id]))
+        &.include?(hearing_judge_id(appeal))
     end
   end
 
-  def hearing_judge_id(uuid)
-    Appeal.find_by(uuid: uuid)&.hearings&.select(&:held?)&.max_by(&:scheduled_for)&.judge_id
-  end
-
   def distributed_cases_tied_to_ineligible_judges
     @appeals.filter_map do |appeal|
-      appeal[:case_id] if VACOLS::CaseDocket.ineligible_judges_sattyid_cache
-        &.include?(hearing_judge_id(appeal[:case_id]))
+      appeal[:case_id] if Rails.cache.fetch("case_distribution_ineligible_judges")&.pluck(:sattyid)&.reject(&:blank?)
+        &.include?(hearing_judge_id(appeal))
     end
   end
 
-  def hearing_judge_id(uuid)
-    Appeal.find_by(uuid: uuid)&.hearings&.select(&:held?)&.max_by(&:scheduled_for)&.judge_id
+  def hearing_judge_id(appeal)
+    if appeal[:docket] == "legacy"
+      user_id = LegacyAppeal.find_by_vacols_id(appeal[:case_id])&.hearings&.select(&:held?)&.max_by(&:scheduled_for)&.judge_id
+      sattyid = VACOLS::Staff.find_by_sdomainid(User.find_by_id(user_id)&.css_id)&.sattyid
```

Comment:
@calvincostaBAH Remove variable declaration and no need to return at line no 111 as last statement will return value like below.
```suggestion
      VACOLS::Staff.find_by_sdomainid(User.find_by_id(user_id)&.css_id)&.sattyid
```